ember_debug/object-inspector.js

```diff
@@ -772,7 +980,15 @@ function customizeProperties(mixinDetails, propertyInfo) {
 
 function getDebugInfo(object) {
   let debugInfo = null;
-  const objectDebugInfo = get(object, '_debugInfo');
+  let objectDebugInfo = get(object, '_debugInfo');
+  if (object instanceof Ember.ObjectProxy) {
+    if (!object.content) {
+      objectDebugInfo = null;
+    }
+    if (object.content) {
+      object = object.content;
```

Comment:
mmm, but for ember data it had issues with displaying data.
the `_debugInfo` is accessed with `get`, which passes through ember proxy.
It's also a bit annoying to click 2 times whenever I want to inspect e.g. `belongsTo` data...
Can we make a special case for ember data models?

Reply:
So, loaded up an Ember app to take a look, `emberclear`, and it looks like currently at least models are already special cased. Whenever I click on a model, and then on a relationship within that model, it shows the model itself, not the proxy first. I think keeping that behavior would be fine since it's already established.

FWIW, I don't necessarily think this would be a bad idea either, but it seems out of scope for this PR. I think we should fix the existing behavior for sure, but any additional proxy changes should be followups.

Reply:
strange, for me it keeps failing in ember-data `_debugInfo`...  version `3.11.4`, but sure, i can extract this parts into another PR